File: SimpleNotification/app/src/main/java/com/dstrube/simplenotification/NotificationReceiverActivity.java
package com.dstrube.simplenotification;

import android.app.Activity;
import android.os.Bundle;

public class NotificationReceiverActivity extends Activity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.notification_alert);
    }
}

File: kotlin/SimpleNotification/app/src/main/java/com/dstrube/simplenotification/NotificationReceiverActivity.kt
package com.dstrube.simplenotification

import android.os.Bundle
import android.app.Activity


class NotificationReceiverActivity : Activity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.notification_alert)
    }
}